learning_resources_search/plugins.py

```diff
@@ -133,11 +146,14 @@ def bulk_resources_unpublished(self, resource_ids, resource_type):
             resource_ids,
             chunk_size=settings.OPENSEARCH_INDEXING_CHUNK_SIZE,
         ):
-            try_with_retry_as_task(
-                tasks.bulk_deindex_learning_resources,
-                ids,
-                resource_type,
-            )
+            unpublished_tasks = [
+                tasks.bulk_deindex_learning_resources.si(ids, resource_type),
+            ]
+            if django_settings.QDRANT_ENABLE_INDEXING_PLUGIN_HOOKS:
```

Comment:
I think the elasticsearch tasks should be chained before the vector embedding tasks since the vector embedding tasks can potentially take a long time. It's better to make the elasticsearch index up to date sooner